modules/datadict/php/datadict.class.inc

```diff
@@ -98,17 +126,16 @@ class Datadict extends \NDB_Menu_Filter
         );
         $this->addSelect('sourceFrom', 'Instruments', $instruments);
         $this->addSelect(
-            'Description',
-            'Description',
+            'descriptionStatus',
+            'Description Status',
             array(
              ''           => 'All',
```

Comment:
can we remove the `'' => All` options for the selects (sourceFrom & Description)? they seem unnecessary

Reply:
@llevitis also applies to the instrument select